Pod/Classes/NBNRealmBrowser.m

```diff
@@ -8,6 +8,16 @@
 
 static NSString *CellIdentifier = @"CellIdentifier";
 
+@interface RLMRealm ()
++ (instancetype)realmWithPath:(NSString *)path
```

Comment:
Exactly 

-Samuel E. Giddins

On Mar 19, 2015, at 12:00 AM, Piet Brauer <notifications@github.com> wrote:

In Pod/Classes/NBNRealmBrowser.m:

> @@ -8,6 +8,16 @@
>  
>  static NSString *CellIdentifier = @"CellIdentifier";
>  
> +@interface RLMRealm ()
> ++ (instancetype)realmWithPath:(NSString *)path
Why should we use a private method here? Because of the dynamic property?

—
Reply to this email directly or view it on GitHub.